.gitignore

```diff
@@ -15,3 +15,4 @@ webdriver/.idea
 .DS_Store
 *.rej
 _venv
+webdriver/.cache
```

Comment:
I could add `*.cache` instead or add it to my system gitignore, seems strange that this would ever be included however?